scripts/clearoutputs.py

```
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+import sys
+from pathlib import Path
+
+current_dir = Path(os.getcwd())
+parent_dir = Path(os.path.realpath(__file__)).parent.parent
+
+if current_dir != parent_dir:
+    raise Exception(f"Script can only be run in {parent_dir}")
```

Ruff (0.12.2)

Create your own exception

(TRY002)